assets/settings/default.json

```diff
@@ -92,8 +92,9 @@
   // Whether to use additional LSP queries to format (and amend) the code after
   // every "trigger" symbol input, defined by LSP server capabilities.
   "use_on_type_format": true,
-  // Whether to automatically type closing characters for you. For example,
-  // when you type (, Zed will automatically add a closing ) at the correct position.
+  // Whether to automatically add matching closing characters when typing
+  // opening parenthesis, bracket, brace, single or double quote characters.
+  // For example, when you type (, Zed will add a closing ) at the correct position.
```

Comment:
Do you also want to update the text here? https://github.com/zed-industries/zed/blob/faebce8cd0314dbcc2279da5d6fa74c8785fa8ab/docs/src/configuring_zed.md#L388-L397

Reply:
Great idea! Done!